src/pages/build-apps/guides/authentication.md

```diff
@@ -0,0 +1,264 @@
+
+![Modal displayed by showConnect function](/images/todos/get-started.png)
+
+The `showConnect` function accepts a number of properties within a parameter object such as:
+
+- The app's `name` and `icon`: provided as strings comprising the `appDetails` object property.
+- The `redirectTo` string: used to provide a URL to which the user should be redirected upon successful authentication. The `finished` callback serves a similar purpose by handling successful authentication within a context of a popup window.
+- The `userSession` object initiated above.
+
+Once the user selects the button presented in this modal, they are passed to the Stacks Wallet for authenticator with the `authRequest` token as a GET parameter. From there they can confirm authentication and generate a new _Secret Key_ or Stacks identity before doing so, as needed before coming back to the app.
+
+## Handle pending authentication
+
+Unless the user has confirmed authentication within the context of a popup window, they will get redirected back to the app via the `redirectTo` address provided above, at which point the app needs to handle the pending authentication state using the `authResponse` value provided as a GET parameter:
+
+```jsx
+import { AppConfig, UserSession, showConnect } from '@stacks/connect';
+
+const appConfig = new AppConfig(['store_write', 'publish_data']);
+const userSession = new UserSession({ appConfig });
+
+window.onload = function () {
+  if (userSession.isSignInPending()) {
+    userSession.handlePendingSignIn().then(userData => {
+      // Save or otherwise utilize userData post-authentication
+    });
+  } else if (userSession.isUserSignedIn()) {
+    // Handle case in which user is already authenticated
+  }
+};
+```
+
+The `isSignInPending` method of the `userSession` object is used to detect whether the user needs to handle a pending authentication state upon page load.
+
+The `handlePendingSignIn` method is then used to handle that state, returning a `userData` object with all the data needed to save the user's information into their session.
+
+The authenticated state can later be detected by the `isUserSignedIn` method in case any particular handling is needed then.
+
+~> Note that implementing `handlePendingSignIn` is especially important for supporting authentication within the context of mobile apps.
+
+If the user has indeed confirmed authentication in the context of a popup window, the authenticator will resolve the pending authentication state automatically with the app within the parent window.
+
+It will then trigger the `finished` function provided above, which can be used similarly to save the user's information into their session as retrieved with `userSession.loadUserData()`.
+
+## Usage in React Apps
+
+Import the `useConnect` from the `connect` package to integrate transaction signing more seamlessly into React apps.
+
+TODO: Add guidance for authentication with React
```

Comment:
@hstove Mind providing code snippet(s) here that illustrate authentication integration for React apps, akin to what we have for tx signing here? https://github.com/blockstack/docs/pull/1014/files#diff-1feffcde7c41bc28d3d1db81bdf7c4ab24d029e76c90f058e4d058024c0ae057R225